src/registrar/fixtures.py

```diff
@@ -116,6 +116,12 @@ class UserFixture:
             "username": "5dc6c9a6-61d9-42b4-ba54-4beff28bac3c",
             "first_name": "David-Analyst",
             "last_name": "Kennedy-Analyst",
+        }
+        {
+           "username": "0eb6f326-a3d4-410f-a521-aa4c1fad4e47",
+           "first_name": "Gaby-Analyst",
+           "last_name": "DiSarli-Analyst",
+           "email": "gaby@truss.works"
```

Comment:
```suggestion
           "email": "gaby@truss.works",
```

There was a linting error because it was missing a comma just for how its processed, but should pass after you add the comma in!